nodes/widgets/ui_template.html

```diff
@@ -70,28 +82,42 @@
 
                 if (typeof this.templateScope === 'undefined') {
                     this.templateScope = 'local'
-                    $('#node-input-templateScope').val(this.templateScope)
+                    $templateScope.val(this.templateScope)
                 }
 
-                $('#node-input-templateScope').on('change', function () {
-                    if ($('#node-input-templateScope').val() === 'global') {
-                        $('#template-row-group, #template-row-size, #template-pass-store, #template-row-class').hide()
-                        that._def.defaults.group.required = false
-                    } else {
-                        $('#template-row-group, #template-row-size, #template-pass-store, #template-row-class').show()
+                $templateScope.on('change', function () {
```

Comment:
`that._def.defaults.group.required = false` disables validation on the `group` config selector - because a site level style above group therefore hide the group select & prevent it informing the editor of any misconfiguration

in short, 
* when option "Widget in group" is chosen, we disable validation and hide the "page" and "site" selection & show the "group" selection
* when option "Site Style" is chosen, we disable validation and hide the "page" and "group" selection & show the "site" selection
* when option "Page Style" is chosen, we disable validation and hide the "site" and "group" selection & show the "page" selection